app/views/layouts/application.html.erb

```diff
@@ -27,16 +28,16 @@
                 <li class="nav-item"><a class="nav-link js-scroll-trigger" href="/quiz">Take the Fear Quiz</a></li>
                 <li class="nav-item"><a class="nav-link js-scroll-trigger" href="/games">Games</a></li>
                 <% if current_user.nil? %>
-                  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#login">Login</a></li>
-                  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#register">Register</a></li>
+                  <li class="nav-item"><a class="nav-link js-scroll-trigger" href="/auth/google_oauth2">Log In with Google</a></li>
```

Comment:
Updated links and removed 'register' since they both register and log in with Google

Reply:
I wonder if after we complete all of our tasks, we could replace the link with an actual google button

![41761606-83b5bd42-762a-11e8-811a-b78fdf68bc04](https://user-images.githubusercontent.com/59414750/88487445-635c3580-cf42-11ea-8410-cfb9f91d0fc5.png)

Reply:
We can definitely add that after functionality is completed. I played around with it yesterday and it just makes the navbar a little funky

Reply:
Thats a great idea